pennylane_lightning/src/util/BitUtil.hpp

```diff
@@ -37,10 +38,9 @@ inline auto constexpr log2PerfectPower(uint64_t val) -> size_t {
 }
 
 /**
- * @brief Check if there is a positive integer n such that value == 2^n.
+ * @brief Compute log2 of value in a compile-time.
```

Comment:
I'm not following the change in `@brief`.
It looks like we are still making a check here.